src/config/2019.json

```diff
@@ -829,6 +829,15 @@
       "github": "noahblon",
       "twitter": "noahblon"
     },
+    "noah-vdv": {
+      "name": "Noah van der Veer",
+      "teams": [
+        "translators"
+      ],
+      "github": "noah-vdv",
+      "linkedin": "Noah Van der Veer",
```

Comment:
Is this correct? Didn't think you could have spaces in your name and https://linkedin.com/in/noah%20van%20der%20veer doesn't work.

Don't need a LinkedIn profile btw if you don't want one.

Reply:
Yeah, I'll take it off